.github/workflows/ci.yml

```diff
@@ -0,0 +1,100 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - 'master'
+
+
+jobs:
+
+  lint:
+    name: Linter
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2.3.4
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Cache pip
+      uses: actions/cache@v2.1.6
+      with:
+        path: ~/.cache/pip
+        key: ${{ hashFiles('**/setup.py') }}-${{ hashFiles('**/requirements*.txt') }}
+    - name: Install libenchant
+      run: sudo apt-get install libenchant-dev
```

Comment:
Do you know what uses this? Normally, it's spellcheck extension for sphinx. I see this job only runs flake8 so it shouldn't be necessary.

Reply:
Yes it's a spellcheck library for sphinx.